lib/Agrammon/DB/User.pm6

```diff
@@ -126,7 +126,7 @@ class Agrammon::DB::User does Agrammon::DB {
                 if not $valid {
                     warn 'PW update failed';
                 }
-                return $valid;
+                return $valid ?? %( :changed(1)) !! False;
```

Comment:
Hm, bit of an odd return value situation here. Would be better as a straight true/false at the DB level?

Reply:
Removed the return value and use 204 instead